unison-src/tests/tictactoe2.u

```diff
@@ -18,36 +30,19 @@ isWin board =
     -- horizontal left/center/right
     -- diagonal rising/falling
     Board a b c
-          _ _ _
```

Comment:
I love the implication that implementing a pattern match checker involves fixing a game of tic-tac-toe 🙃

Reply:
Yeah, the first case is the only non-redundant case in this one. The author meant to be using `same` as a guard it looks like.